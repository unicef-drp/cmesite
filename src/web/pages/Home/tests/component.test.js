import React from 'react';
import { shallow } from 'enzyme';
import Component from '../component';

const props = {
  posts: [{ id: 1, content: { rendered: 'rendered' } }],
};

describe('app |Â pages | LandingPage | component', () => {
  describe('render', () => {
    it('should match snapshot', () => {
      const wrapper = shallow(<Component {...props} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
