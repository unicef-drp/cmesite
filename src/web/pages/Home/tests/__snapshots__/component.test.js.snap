// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app |Â pages | LandingPage | component render should match snapshot 1`] = `
<HomePage
  classes={
    Object {
      "wrapper": "HomePage-wrapper-1",
    }
  }
  posts={
    Array [
      Object {
        "content": Object {
          "rendered": "rendered",
        },
        "id": 1,
      },
    ]
  }
/>
`;
